presto-hive/src/test/java/com/facebook/presto/hive/TestCoercingFilters.java

```diff
@@ -70,4 +79,41 @@ public void testVarcharToInteger()
 
         assertFalse(coercingFilter.testNull());
     }
+
+    private final TypeManager typeManager = MetadataManager.createTestMetadataManager().getFunctionAndTypeManager();
+
+    @Test
+    public void testShortToInteger()
```

Comment:
I think it's ok to put all these test cases into one test, but it's up to you.

Reply:
Putting them in separate tests decouples them so they can pass or fail and be debugged independently of each other.